CHANGELOG.md

```diff
@@ -1,3 +1,31 @@
+## 41.0
+- Remove support for file path breakpoints; handle multiple vm breakpoints better (#4019)
+- Force Android Studio to show the Project view (#4031)
+- Build for ADS4 canary 1 (#4017)
+- Check for project disposed during format on save (#4022)
+- Remove an Observatory link handler from DartVmServiceDebugProcess.java (#4018)
+- Detect whether the VM service connection supports the getMemoryUsage API (#4016)
+- Don't invoke eval when shutting down (#4015)
+- Re-enable our disabled tests (#4013)
+- Co-edit module created in Android Studio (#4004)
+- Address several warnings in the IntelliJ log about leaked resources (#3999)
+- Remove an unused method (#4009)
+- Update CONTRIBUTING.md
+- Change the inspector tool window to not only show when an app is running (#4000)
+- Remove the Performance tab from the Inspector; remove Observatory actions (#4008)
+- Hide the custom logging view setting (#4005)
+- Fix a project assert in the Flutter editor notification (#3994)
+- Address a few warnings about 'mostly idle daemon process' (#3995)
+- Add more info to eval() exceptions (#3992)
+- Fix an NPE when switching platforms (#3990)
+- Fix an NPE in the feedback button (#3989)
+- Rename two wrap with Flutter actions (#3988)
+- Remove support for the older _Logging event (#3986)
```

Comment:
We'll need to escape the markdown here somehow; perhaps adding single quotes around it would be enough.

Reply:
Yes, done.